src/components/HTMLEngineProvider/htmlEngineUtils.ts

```diff
@@ -7,12 +7,9 @@ const MAX_IMG_DIMENSIONS = 512;
  * is used by the HTML component in the default renderer for img tags to scale
  * down images that would otherwise overflow horizontally.
  *
- * @param {string} tagName - The name of the tag for which max width should be constrained.
- * @param {number} contentWidth - The content width provided to the HTML
- * component.
- * @returns {number} The minimum between contentWidth and MAX_IMG_DIMENSIONS
+ * @returns The minimum between contentWidth and MAX_IMG_DIMENSIONS
```

Comment:
It's okay to leave the description of the parameter, just skip the type
```suggestion
 * @param tagName - The name of the tag for which max width should be constrained.
 * @param contentWidth - The content width provided to the HTML
 * component.
 * @returns The minimum between contentWidth and MAX_IMG_DIMENSIONS
```